app/defines/interrupt_handler.h

```diff
@@ -31,7 +31,7 @@ void interrupt_handler() {
         csr_mstatus  = read_mstatus();     
         mie_bit  = (csr_mstatus >> 3) & 0x1;                // read mie bit from mstatus. mie = mstatus[3]
```

Comment:
From what i remember - this is the correct coding style.
at some point, we should have SW expert give the C code a look..

I do know someone who may be able to assist here and give his notes on the low-level C code we have, linker, etc.